src/api/forms/service.js

```diff
@@ -189,6 +189,56 @@ export async function updateDraftFormDefinition(formId, definition, author) {
   }
 }
 
+/**
+ * @param {string} formId - ID of the form
+ * @param {Partial<FormMetadataInput>} formUpdate - full JSON form definition
+ * @returns {Promise<string>}
+ */
+// TODO: author to be used to update updatedBy in next PR
+export async function updateFormMetadata(formId, formUpdate) {
+  logger.info(`Updating form metadata (draft) for form ID ${formId}`)
```

Comment:
`(draft)` can be removed from here, since these changes do not apply to the draft and live.